lxd/storage_ceph.go

```diff
@@ -2648,3 +2682,111 @@ func (s *storageCeph) GetStoragePoolVolume() *api.StorageVolume {
 func (s *storageCeph) GetState() *state.State {
 	return s.s
 }
+
+func (s *storageCeph) StoragePoolVolumeSnapshotCreate(target *api.StorageVolumeSnapshotsPost) error {
```

Comment:
The ceph codepath is the one I'm most worried. It should be ensured that we have sufficient tests that we don't run into any errors or leave dead volumes in the osd pools.